### PR TITLE
side-bar now gets focused on dragging and dropping text

### PR DESCRIPTION
**Pull request by himanish-star**
fixes: #518 .

I have fixed this. But the colour combinations may not be the best of all. So if you suggest me then surely I will change the colours accordingly.

The GIF below shows the changes made :)
ezgif com-video-to-gif

### CODE REVIEW

**Comment by himanish-star** [comment]
@Natim, @vladikoff . Could you review this PR of mine

**Comment by Natim** [comment]
> @Natim, @vladikoff . Could you review this PR of mine

Take your time and breathe, we are not working during week-ends so opening a PR is sufficient no need to ping us multiple times per issues.

**Comment by Natim** [comment]
This PR looks good to me, @ryanfeeley can you validate that the color code is correct in your opinion?

**Comment by himanish-star** [comment]
> Take your time and breathe, we are not working during week-ends so opening a PR is sufficient no need to ping us multiple times per issues.

I'm sorry. This will never happen again. I'm sorry for the inconvenience caused

**Comment by ryanfeeley** [comment]
Ths is great @himanish-star! Much better than what other text areas do (like even this one in GitHub). Most just do the focus outline. I like the light BG colour and the coloured outline.

File: src/sidebar/editor.js [hunk 1]
```diff
@@ -15,9 +15,24 @@ function customizeEditor(editor) {
...
+  
+  document.addEventListener('dragover', () => {
+    const mainEditor = document.querySelector('.ck-editor__main');
+    mainEditor.style.border = '1px solid green';
+    mainEditor.style.backgroundColor = '#dcd6da';
+  });
+
+  document.addEventListener('dragleave', () => {
+    const mainEditor = document.querySelector('.ck-editor__main');
+    mainEditor.style.border = 'none';
```
Comment (Natim) [inline src/sidebar/editor.js]: Shall we use inherit here too?

**Comment by Natim** [comment]
Thanks a lot @himanish-star

**Pull request merged**